src/core/meshcop/commissioner.cpp

```diff
@@ -284,7 +284,7 @@ otError Commissioner::AddJoiner(const Mac::ExtAddress *aEui64, const char *aPskd
 
     VerifyOrExit(StringLength(aPskd, Dtls::kPskMaxLength + 1) <= Dtls::kPskMaxLength, error = OT_ERROR_INVALID_ARGS);
 
-    RemoveJoiner(aEui64, 0); // remove immediately
+    RemoveJoiner(aEui64, 0, /* aNotifyLeader */ false); // remove immediately
```

Comment:
nit: generally prefer having an enum rather than bool here to avoid the need for having the comment

Reply:
revised the code to use `enum JoinerOperationFlag`, please check